packages/modals/examples/drawer-modal.md

```diff
@@ -0,0 +1,68 @@
+```jsx
+const { Button } = require('@zendeskgarden/react-buttons/src');
+const { Paragraph } = require('@zendeskgarden/react-typography/src');
+
+const Example = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const open = () => setIsOpen(true);
+  const close = () => setIsOpen(false);
+
+  return (
+    <>
+      <Button onClick={open}>Open Drawer</Button>
+      <DrawerModal isOpen={isOpen} onClose={close}>
+        <DrawerModal.Header>Example Title</DrawerModal.Header>
+        <DrawerModal.Close />
```

Comment:
For other Modal examples we've included the `Close` component at the end of the tab order. Should we do the same in this example?

Reply:
Yeah that sounds good.